core/src/main/java/org/aerogear/mobile/core/metrics/publisher/NetworkMetricsPublisher.java

```diff
@@ -56,12 +54,9 @@ public void publish(final Metrics... metrics) {
 
             final HttpResponse httpResponse = httpRequest.execute();
             httpResponse.onSuccess(() -> {
-                Logger logger = MobileCore.getLogger();
-                logger.debug(MessageFormat.format("Metrics response: {0}: {1}",
-                                httpResponse.getStatus(), httpResponse.stringBody()));
-                logger.debug("Metrics sent: " + json.toString());
+                listener.onPublishMetricsSuccess(httpResponse);
```

Comment:
I think here the lib can check if it was an HTTP_OK (Status 200) and don't need to expose the response.

Reply:
Fair enough